frontend/tibiaguess-frontend/src/components/Navbar.js

```diff
@@ -1,17 +1,29 @@
 import Container from 'react-bootstrap/Container';
 import Navbar from 'react-bootstrap/Navbar';
 
-
 function BrandBar() {
   return (
     <>
-      <Navbar>
+      {/* <Navbar>
         <Container>
           <Navbar.Brand href="#home">Tibiaguessr</Navbar.Brand>
         </Container>
-      </Navbar>
+      </Navbar> */}
+      <nav>
+        <div className='nav--social--wrapper'>
+          <i class='fa-brands fa-github fa-2x'></i>
+          <i class='fa-brands fa-tiktok fa-2x'></i>
+          <i class='fa-brands fa-instagram fa-2x'></i>
+          <i class='fa-brands fa-twitter fa-2x'></i>
+        </div>
+        <div className='nav--items'>
```

Comment:
text and logo are not centered in the middle of the screen but centered in the div resulting it being misaligned